packages/togetherai/src/togetherai-provider.ts

```diff
@@ -86,6 +92,14 @@ Creates a text embedding model for text generation.
     modelId: TogetherAIEmbeddingModelId,
     settings?: TogetherAIEmbeddingSettings,
   ): EmbeddingModelV1<string>;
+
+  /**
+   * Creates a model for image generation.
+   */
+  image(
+    modelId: TogetherAIImageModelId,
+    settings?: TogetherAIImageSettings,
+  ): ImageModelV1;
```

Comment:
Add `imageModel` to support the provider registry / custom providers (see Provider v1 spec).